client/app/components/packages/attachments.hbs

```diff
@@ -0,0 +1,156 @@
+<h3>
+  Attached Documents
+  <a
+    href="#"
+    class="section-anchor"
+    id="uploads"
+    data-magellan-target="uploads"
+  >
+  </a>
+</h3>
+
+<p>
+  Please attach the following required documents. (Additional materials that help to explain the proposed project may also be submitted.)  
+</p>
+
+<div class="site-main grid-x grid-margin-x">
+  <div class="cell small-6">
+    <b>REQUIRED FOR ALL PROJECTS:</b>
+    <ul>
+      <li><a href="#">Tax Map(s)</a> — if project area has more than one tax block submit a separate map for each tax block</li>
+      <li><a href="#">Official Zoning Map</a> — circle the project area</li>
+      <li><a href="#">Land Use Map</a></li>
+      <li>Site Plan — include building heights, unit counts and square footage calculations (GSF &nbsp; ZSF) for proposed project (certain exceptions apply, consult the Lead Planner)</li>
+      <li>Photographs</li>
+      <li>Signature Form</li>
+    </ul>
+  </div>
+  <div class="cell small-6">
+    <b>FOR PROJECTS IN THE NYC COASTAL ZONE:</b>
+    <ul>
+      <li>Map of <a href="#">project location within the NYC coastal zone</a> including Special Area Designations</li>
+    </ul>
+    <b>FOR <a href="#">CITY MAP CHANGES</a>:</b>
+    <ul>
+      <li>Most recent Borough President’s final Section Map or most recent approved and filed Alteration Map for the project area</li>
+      <li>Site survey — recommended but not required</li>
+    </ul>
+  </div>
+  <div class="cell">
```

Comment:
All this stuff doesn't need to be in the component. It can go in the `<PasForm >` component (I've already got it in my PR — https://github.com/NYCPlanning/labs-applicant-portal/pull/109). This component only needs to be the part indicated in the wireframes by the gray line: 

![image](https://user-images.githubusercontent.com/409279/80650178-7cd40b00-8a41-11ea-8cd5-703e1b702fd3.png)

Reply:
Cool, thx I will remove!